File: week1/Day2/ExerciseXP/exercise4.py
#numbers = (1.5, 2, 2.5, 3, 3.5, 4, 4.5, 5.)
numbers = 1.5
while numbers < 5.5:
    if numbers.is_integer():
        print(int(numbers))
    else:
        print(numbers)
    numbers +=0.5

File: Week5/Day2/exercise4.js
// Create a class Calculator with the following static methods:
// add(a: number, b: number): number – returns the sum of two numbers.
//     subtract(a: number, b: number): number – returns the difference between two numbers.
// Call these methods without creating an instance of the class.
var Calculator = /** @class */ (function () {
    function Calculator() {
    }
    Calculator.suma = function () { return this.a + this.b; };
    ;
    Calculator.subtract = function () { return this.a - this.b; };
    ;
    return Calculator;
}());
Calculator.a = 7;
Calculator.b = 32;
console.log(Calculator.suma());
console.log(Calculator.subtract());
